### DISCUSSION TITLE
Problem with WebKitGTK

### DISCUSSION THREAD

**Discussion by lucascompython** [tauri-apps/tauri#9088] [Q&A]
Hi.
On Linux Tauri uses WebKitGTK, and I've noticed some problems with it.

First is, when a CSS animation is playing, the rest of the app gets blurry.

And the other problem is with spans that are contenteditable="true". The expected behavior is for it to just act as an input element, but I have to right-click for it to work.

From my testing, this also appears to be a problem in the Epiphany Browser that uses a more recent version of WebKitGTK.

Are there any workarounds for this? 

Here is a video showing the problems: [https://imgur.com/pvmx8nc](https://imgur.com/pvmx8nc)

**Comment by novecentonove** [comment]
I have the same problem, only now, that i'm switched to Ubuntu 24.04. 
I'm using Tauri with Vite and TipTap js.
